Dockerfile

```diff
@@ -3,28 +3,23 @@ FROM ubuntu:18.04
 LABEL mantainer="Read the Docs <support@readthedocs.com>"
 LABEL version="5.0.0rc1"
 
-ENV DEBIAN_FRONTEND noninteractive
 ENV APPDIR /app
 ENV LANG C.UTF-8
 
-# Versions, and expose labels for external usage
-ENV PYTHON_VERSION_27 2.7.15
-ENV PYTHON_VERSION_36 3.6.8
-ENV PYTHON_VERSION_37 3.7.2
-ENV CONDA_VERSION 4.5.12
-LABEL python.version_27=$PYTHON_VERSION_27
-LABEL python.version_36=$PYTHON_VERSION_36
-LABEL python.version_37=$PYTHON_VERSION_37
-LABEL conda.version=$CONDA_VERSION
+# UID and GID from readthedocs/user
+RUN groupadd --gid 205 docs \
+ && useradd  --gid 205 -m --uid 1005 docs
 
 # System dependencies
-RUN apt-get -y update
-RUN apt-get -y install \
+RUN export DEBIAN_FRONTEND=noninteractive \
```

Comment:
I think we don't need `export` here, since it's being preprended to all commands.

Reply:
Sorry, no. It's necessary.

To make it clear, here's a small example:

```
[bebe:~] % VAR=int
[bebe:~] % echo $VAR                            
int
[bebe:~] % printenv | grep VAR            
[bebe:~] % VAR=glob printenv | grep VAR
VAR=glob
[bebe:~] % VAR=glob && printenv | grep VAR
[bebe:~] % export VAR=exported && printenv | grep VAR
VAR=exported
```

The difference is exactly in the last two prompts.